components-testing/src/styles.css

```
@@ -0,0 +1,31 @@
+/* Import bundled styles from components package */
+@import '@opentrons/components/styles';
+
+.app_container {
+  padding: 1rem;
+}
+
+.analysis_info {
+  padding: 1rem;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  background-color: #f0f0f0;
+}
+
+.protocol_deck_container {
+  overflow: hidden;
+  width: 1200px;
```

nit: very nitpicky but we try to make sure everything is in rem - you can just divide all the px values by 16! 
```suggestion
  width: 75rem;
```

---

fixed them all